src/ufo-general-backproject-task.c

```diff
@@ -518,7 +518,7 @@ make_template (UfoGeneralBackprojectTaskPrivate *priv, gboolean vectorized)
         return NULL;
     }
     if (vectorized) {
-        if (!priv->parameter == PARAMETER_Z) {
+        if (!(priv->parameter == PARAMETER_Z)) {
```

Comment:
Thanks.

Reply:
Still a stupid fix, the most straightforward and easy to read one would be `if (priv->parameter != PARAMETER_Z)` :smile:

Reply:
Actually... :smile: